pyiron_continuum/damask/damaskjob.py

```diff
@@ -20,7 +20,8 @@ def __init__(self, project, job_name):
         self.input['grid'] = np.array([16,16,16])
         self.input['size'] = np.array([1.0,1.0,1.0])
         self.input['grains'] = 20
-        self.executable = "DAMASK_grid -l tensionX.load -g damask.vtr"
+        self.input['cores'] = 4
+        self.executable = "mpiexec -np "+str(self.input['cores'])+" DAMASK_grid -l tensionX.load -g damask.vtr"
```

Comment:
```suggestion
        self._executable_activate()
```

Reply:
The executable activate script searches for shell scripts to link to the executable.

Reply:
Shouldn't we add DAMASK to the resources folder and 'just' use our normal setup for executables? I opened a PR to do so [here](https://github.com/pyiron/pyiron-resources/pull/41). Maybe, @Ahmed-Aslam could you add the version number to the executable name like we usually do that?